doc/api/fs.md

```diff
@@ -1304,7 +1304,11 @@ changes:
 * `options` {string|Object}
   * `encoding` {string} **Default:** `'utf8'`
   * `withFileTypes` {boolean} **Default:** `false`
-  * `recursive` {boolean} **Default:** `false`
+  * `recursive` {boolean|Function} **Default:** `false`
+    * `name` {string} A given directory file system artifact name.
```

Comment:
```suggestion
    * `name` {string|fs.Dirent} A given directory file system artifact name or the `Dirent` instance associated with it.
```

Reply:
Or also it can be `Buffer`, depending on `encoding` option
```suggestion
    * `name` {string|Buffer|fs.Dirent} A given directory file system artifact name or the `Dirent` instance associated with it.
```